bika/lims/browser/analysisrequest/templates/analysisrequest_retract_mail.pt

```diff
@@ -4,7 +4,7 @@
         portal portal_url/getPortalObject;
         ar python:view.context;
         rej_widg    python:ar.getRejectionReasons();
-        reasons     python:ar.getRejectionReasons()[0].get('selected',[]) if rej_widg else [];
+        reasons     python:rej_widg[0].get('selected', [])+([rej_widg[0].get('other', '')] if rej_widg[0]['checkbox_other'] else []) if rej_widg else [];
```

Comment:
Maybe more understandable and consistent if you rephrase this definition like follows:

```
reasons  python: view.get_rejection_reasons();
```
and add the counterpart functions in [reject.py](https://github.com/senaite/senaite.core/blob/master/bika/lims/browser/analysisrequest/reject.py):

```python
def get_rejection_reasons(self, keyword=None):
    """Returns a list with the rejection reasons as strings"""
    keys = ['selected', 'other']
    if keyword is None:
        return map(self.get_rejection_reasons, keys)
    if keyword not in keys:
        return []
    rejection_reasons = self.context.getRejectionReasons()
    rejection_reasons = rejection_reasons and rejection_reasons[0] or {}
    return rejection_reasons.get(keyword, [])
```